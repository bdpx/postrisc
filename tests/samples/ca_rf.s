.text
    alloc	96
    write	"test ca.rf"
    ca.rf	%r22, ca_rf_data
    write	"ca.rf: %x64(r22)"

    write	"end_ca_rf_test"
.data
ca_rf_data:

.end
