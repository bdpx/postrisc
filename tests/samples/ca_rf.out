test ca.rf
ca.rf: fffffffff1250000
end_ca_rf_test
